Mathlib/Algebra/Group/Subgroup/Lattice.lean

```diff
@@ -470,6 +470,18 @@ theorem closure_eq_top_of_mclosure_eq_top {S : Set G} (h : Submonoid.closure S =
     closure S = ⊤ :=
   (eq_top_iff' _).2 fun _ => le_closure_toSubmonoid _ <| h.symm ▸ trivial
 
+theorem toAddSubgroup_closure (S : Set G) :
```

Comment:
Could you add lemmas for `toAddSubgroup'` and `toSubgroup'` as well?

Reply:
Ok, it's done.